tests/build/_internal/test_cmake_file.py

```diff
@@ -2,63 +2,58 @@
 # Copyright (C) 2020 Arm Mbed. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 #
+import pytest
+
 from tests.build._internal.config.factories import ConfigFactory
 from mbed_tools.build._internal.cmake_file import generate_mbed_config_cmake_file, _render_mbed_config_cmake_template
 
 
+toolchain_name = "gcc"
```

Comment:
We should use all caps with underscores for global constants.

Reply:
OK